src/angular/date-input/date-input.ts

```diff
@@ -56,7 +56,7 @@ export class SbbDateInput<T = Date> extends SbbControlValueAccessorMixin(class {
     return this.#element.nativeElement.max;
   }
 
-  @Input()
+  @Input({ alias: 'date-filter' })
```

Comment:
Does the eslint stuff not see that? do we have maybe more similar problems?

Reply:
That the alias is not set if it has `attribute: false`; 
I noticed that the same property in the datepicker has the attribute set, so i took from there. 

I don't remember the reason why i put it in the datepicker, but i see that using [dateFilter]=${dateFilter} in the story throws an error.

Reply:
which error happens?

Reply:
I noticed that removing the `argsToTemplate` and passing the `dateFilter` with square brackets it breaks with or without the `data-filter` alias; so probably is an issue with Storybook (?!). The error I receive is:
![image](https://github.com/user-attachments/assets/c0a680cc-4f7d-4f2c-bd7e-21a05006741c)

Reply:
SOLUTION:
remove the `alias` property for inputs in the whole repo.

Since this change is unrelated to this story and storybook have to be checked for regressions, i'm resolving this conversation, so the date-input fix can be merged;
I'm also going to open a new story for the 'alias' removal.